mods/d2k/rules/arrakis.yaml

```diff
@@ -87,7 +87,7 @@ sandworm:
 		Speed: 42
 		Locomotor: worm
 	Targetable:
-		TargetTypes: Ground, Creep
+		TargetTypes: Ground, Creep, IgnoredByBots
```

Comment:
I can see the reasoning behind this, but Sandworms are the only D2k actor with `Creep` target type, so in my opinion just use that (I doubt we'll want D2k AI to ignore any other actors anytime soon, and we can still introduce `IgnoredByBots` when that happens).

Reply:
Done.